java/org/apache/catalina/session/StandardSession.java

```diff
@@ -78,6 +81,7 @@
  * @author Craig R. McClanahan
  * @author Sean Legassick
  * @author <a href="mailto:jon@latchkey.com">Jon S. Stevens</a>
+ * @author Carsten Klein
```

Comment:
Please remove. (And all other added `@author` tags).

Reply:
There is a new PR #2. All these issues have already been fixed.

Reply:
I messed up my branch this morning and had to setup a new one. Still a bit new to GIT...